src/network/httpRequest.js

```diff
@@ -1,33 +1,15 @@
-import fetch from 'node-fetch';
+import axios from 'axios';
+import http from 'http';
+import https from 'https';
 
-/**
- * Executes an HTTP request.
- * @param url {String} The URL we want to request.
- * @param options {Object} The fetch options.
- * @return {Object} The response data as a Promise.
- */
-export function request(url, options) {
-  return fetch(url, options)
-    .then(checkStatusCode)
-    .then(parseResponse);
-}
+let instance;
 
-function checkStatusCode(response) {
-  if (response.status >= 200 && response.status < 300) {
-    return Promise.resolve(response);
-  }
-  return Promise.reject(new Error(response.statusText));
-}
-
-/**
- * Parses the HTTP response returned by a network request.
- * @param response {Object} A response from a network request.
- * @return {Object} The parsed JSON from the request.
- */
-function parseResponse(response) {
-  return response.json()
-    .then((res) => res.result || res)
-    .catch((err) => {
-      throw new Error(err);
+export default function getAxios() {
```

Comment:
Can we export this as an object instead of a function so we can use it like
```js
import axios from 'httpRequest';
axios.get(Routes.whatever)
```
also, we might be able to do
```js
import { get } from 'httpRequest';
get(Routes.whatever)
```
Not sure on that last one 100% though^

Also, as far as I know, I don't think the `instance` variable is necessary. Unless you're globally attaching `instance`, each time you import `getAxios`, `instance === undefined` always.
```js
export default axios.create({
  ...
});
```

Reply:
fixed to export axios object.